Modules/Filtering/LabelMap/include/itkAttributeLabelObject.h

```diff
@@ -156,7 +156,7 @@ class AttributeLabelObject : public LabelObject<TLabel, VImageDimension>
   }
 
 private:
-  AttributeValueType m_Attribute;
+  AttributeValueType m_Attribute = 0.0;
```

Comment:
Glad that you found this bug, @jhlegarreta !  I like your suggestion to fix it by an in-class member initialization.

Instead of ` = 0.0`, I would prefer a pair of empty braces, `{}`. Because `AttributeValueType` may not be a floating point type. It may even be a string type (if I understand correctly). So my suggestion is:

    AttributeValueType m_Attribute{};

Reply:
:+1: Changed in 8afd3d5.